include/avif/avif.h

```diff
@@ -747,7 +747,9 @@ struct avifDecoderData;
 
 typedef enum avifDecoderSource
 {
-    // If a moov box is present in the .avif(s), use the tracks in it, otherwise decode the primary item.
+    // Honor the major brand signaled in the beginning of the file to pick between an AVIF sequence
+    // ('avis', tracks-based) or a single image ('avif', item-based). If the major brand is neither
+    // of these, prefer the AVIF sequence ('avis', tracks-based), if present.
```

Comment:
Joe: Chrome's [avif_image_decoder.cc](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/platform/image-decoders/avif/avif_image_decoder.cc) has the following code:

```
    // TODO(wtc): Currently libavif always prioritizes the animation, but that's
    // not correct. It should instead select animation or still image based on
    // the preferred and major brands listed in the file.
    if (animation_option_ != AnimationOption::kUnspecified &&
        avifDecoderSetSource(
            decoder_.get(),
            animation_option_ == AnimationOption::kPreferAnimation
                ? AVIF_DECODER_SOURCE_TRACKS
                : AVIF_DECODER_SOURCE_PRIMARY_ITEM) != AVIF_RESULT_OK) {
      return false;
    }
```

Once Chrome picks up this pull request, I can delete that TODO comment (but leave the code unchanged). Correct?

Reply:
Hmm. I'm not familiar with `AnimationOption`, but it seems like this code would still be useful, no?

When `animation_option_` is set to `kUnspecified`, you probably want the AUTO behavior (don't bother calling `avifDecoderSetSource()`, leave it defaulted to AUTO). If it is specified, perhaps you still want to force this?

Either way, the TODO comment can probably go away. AUTO will now leverage the major brand, which I believe satisfies the TODO.

Reply:
Joe: Thanks for your reply. I agree with you fully.

I am sorry that my question was not clear. I was suggesting that *only* the TODO comment be deleted. I think the code should be left unchanged. What makes the TODO comment confusing is that it is actually a to-do for libavif, not for Chrome.